FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -46,8 +47,17 @@ protected function tearDown()
     private function createTestProduct(array $testProductConfiguration)
     {
         try {
-            /** @var ArticleResource $resource */
-            $resource = Manager::getResource('Article');
+            $manager = Shopware()->Models();
+
+            $resource = new ArticleResource();
+            $resource->setManager($manager);
```

Comment:
Shouldn't this happen after `$manager->create()`?

Reply:
Doesn't really matter as instances are passed by reference and not by value.